_data/locales/zh-cn.yml

```diff
@@ -5,11 +5,11 @@ zh-cn:
     contribute: 贡献
   index:
     lead: 开放源代码软件就是由你这样的人所打造，这里只是为了节省你的学习时间而设，从这里开启你的开源项目之旅吧！
-    opensourcefriday: It's Friday! Invest a few hours contributing to the software you use and love
+    opensourcefriday:又是星期五！投入几个小时为您使用和喜爱的软件做出贡献
```

Comment:
Missing space after `:` https://travis-ci.org/github/opensource.guide/builds/410084030#L1348

Reply:
Alright. Thanks for pointing out my error.